From 765a605ec4c5b2d48fba44846cbe8950f428104d Mon Sep 17 00:00:00 2001
From: Michael Haubenwallner <michael.haubenwallner@ssi-schaefer.com>
Date: Mon, 10 Apr 2017 13:10:39 +0200
Subject: [PATCH] build: Add -no-undefined libtool flag

We have no need for undefined symbols, so we can let the shared
library build even on systems without support for undefined symbols.

[guillem@hadrons.org:
 - Place the option on its own line.
 - Slightly reword commit message. ]

Signed-off-by: Guillem Jover <guillem@hadrons.org>
---
 src/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/Makefile.am b/src/Makefile.am
index f2209e6..5fdcb8f 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -23,6 +23,7 @@ libmd_la_helper_sources = \
 	$(nil)
 
 libmd_la_LDFLAGS = \
+	-no-undefined \
 	-Wl,--version-script=$(srcdir)/libmd.map \
 	-version-number $(LIBMD_ABI)
 libmd_la_DEPENDENCIES = \
-- 
2.14.1

